pkg/util/util.go

```diff
@@ -0,0 +1,207 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build e2e
+// +build e2e
+
+package util
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/sigstore/rekor/pkg/generated/models"
+)
+
+var (
+	cli         = "rekor-cli"
+	server      = "rekor-server"
+	nodeDataDir = "node"
+)
+
+func init() {
+	p := os.Getenv("REKORTMPDIR")
+	if p != "" {
+		cli = path.Join(p, cli)
+		server = path.Join(p, server)
+	}
+}
+
+func OutputContains(t *testing.T, output, sub string) {
+	t.Helper()
+	if !strings.Contains(output, sub) {
+		t.Errorf("Expected [%s] in response, got %s", sub, output)
+	}
+}
+
+func Run(t *testing.T, stdin, cmd string, arg ...string) string {
+	t.Helper()
+	arg = append([]string{coverageFlag()}, arg...)
+	c := exec.Command(cmd, arg...)
+	if stdin != "" {
+		c.Stdin = strings.NewReader(stdin)
+	}
+	if os.Getenv("REKORTMPDIR") != "" {
+		// ensure that we use a clean state.json file for each Run
+		c.Env = append(c.Env, "HOME="+os.Getenv("REKORTMPDIR"))
+	}
+	b, err := c.CombinedOutput()
+	if err != nil {
+		t.Log(string(b))
+		t.Fatal(err)
+	}
+	return stripCoverageOutput(string(b))
+}
+
+func RunCli(t *testing.T, arg ...string) string {
+	t.Helper()
+	arg = append([]string{coverageFlag()}, arg...)
```

Comment:
you don't need to prepend coverageFlag here since this calls Run which prepends it

Reply:
👍